package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@ssense/sscheduler": "^1.3.2",
```

Comment:
probably uninstall this package before the next commit/PR as we will not need it